app/src/main/java/org/mozilla/fenix/utils/ClipboardHandler.kt

```diff
@@ -63,6 +60,13 @@ class ClipboardHandler(val context: Context) {
         }
     }
 
+    /**
+     * We cannot rely on `isPrimaryClipEmpty()` since it triggers a clipboard access notification.
+     */
```

Comment:
```suggestion
    /**
     * Returns whether or not the clipboard data contains text.
     * We cannot rely on `isPrimaryClipEmpty()` since it triggers a clipboard access notification.
     */
```

Let's make sure to add a line in the kDoc about what the function is doing.